### ISSUE TITLE
single GPU is better?

### ISSUE THREAD

**Issue by kyoguan** [Kyubyong/tacotron#56]
<img width="1007" alt="6-gpu" src="https://user-images.githubusercontent.com/29282500/27528645-e91d4132-5a83-11e7-84df-0bbd5402238e.png">
 batch_size = 32*6  and num_gpus = 6(train_multi_gpus.py)

<br/>
<img width="980" alt="1-gpu" src="https://user-images.githubusercontent.com/29282500/27528631-ddb95cf4-5a83-11e7-81fc-c187d50068e6.png">
 batch_size = 32 (train.py)

it seems the single gpu is mush faster and the training curve is smoother.

**Comment by candlewill** [comment]
@kyoguan The loss cure is pretty good. How many training data did you use?

**Comment by kyoguan** [comment]
just use the WEB data set, download from https://dl.dropboxusercontent.com/u/42868014/WEB.zip

**Comment by jarheadfa** [comment]
@kyoguan - Is this the epoch loss or the batch loss?
